src/components/NavigationBar.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react'
```

Comment:
Is this component used? I think it was deleted some time ago and replaced with BottomNavigation. Can you check that it's no longer used in the most current version and delete it if it isn't?